From 5f2f17a488aba4319b537aed040ea13607af128b Mon Sep 17 00:00:00 2001
From: Greg Turner <gregturner@ti.com>
Date: Thu, 17 May 2012 14:25:40 -0500
Subject: [PATCH 1/8] am33x: Add memory addresses for crypto modules

* Add base memory addresses to the am33xx.h header file

These addresses are for the HW crypto modules including TRNG, AES, and SHA/MD5

Signed-off-by: Greg Turner <gregturner@ti.com>
---
 arch/arm/plat-omap/include/plat/am33xx.h |   11 +++++++++++
 1 files changed, 11 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 arch/arm/plat-omap/include/plat/am33xx.h

diff --git a/arch/arm/plat-omap/include/plat/am33xx.h b/arch/arm/plat-omap/include/plat/am33xx.h
old mode 100644
new mode 100755
index a16e72c..96ab1c3
--- a/arch/arm/plat-omap/include/plat/am33xx.h
+++ b/arch/arm/plat-omap/include/plat/am33xx.h
@@ -65,6 +65,17 @@

 #define AM33XX_ELM_BASE		0x48080000

+/* Base address for crypto modules */
+#define AM33XX_SHA1MD5_S_BASE	0x53000000
+#define AM33XX_SHA1MD5_P_BASE	0x53100000
+
+#define	AM33XX_AES0_S_BASE	0x53400000
+#define	AM33XX_AES0_P_BASE	0x53500000
+#define	AM33XX_AES1_S_BASE	0x53600000
+#define	AM33XX_AES1_P_BASE	0x53700000
+
+#define	AM33XX_RNG_BASE		0x48310000
+
 #define AM33XX_ASP0_BASE	0x48038000
 #define AM33XX_ASP1_BASE	0x4803C000

--
1.7.0.4
